code-challenges/filter/challenges-08.test.js

```diff
@@ -0,0 +1,277 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named oddValues that, given an array of integers as input, uses filter to return an array containing only the odd integers.
+
+For example, oddValues([1,2,3]) returns [1,3].
+------------------------------------------------------------------------------------------------ */
+
+const oddValues = (arr) => {
+  let temp = arr.filter((num)=>{
+    return (!(num % 2 === 0));
+  });
+  return temp;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named filterStringsWithVowels that, given an array of strings as input, uses filter to return an array with only words that contain vowels.
+
+The callback function to filter should include or utilize a regular expression pattern.
+
+For example, filterStringsWithVowels('gregor','hound','xyz') returns ['gregor', 'hound'].
+------------------------------------------------------------------------------------------------ */
+
+
+const filterStringsWithVowels = (arr) => {
+  let temp = arr.filter(text=>text.match(/[a,e,i,o,u]/));
```

Comment:
When using `[]` for regex you don't need commas:
`[aeiou]` would have worked. I worry that since you have commas in there, your regex will also return string that have a `,` in them.